From 6f4c0faf1f85132734e94aa39c67143f823b9324 Mon Sep 17 00:00:00 2001
From: Warren <wyun2006@gmail.com>
Date: Sat, 10 Oct 2026 14:07:36 -0400
Subject: [PATCH] Mountain Dew Model (#39047)

* big model: desires-final (ea1600cb/12864)

* big model: mountain dew (870a4823/1284)

* big model: MD retrained (8d736683/12864)

* rebase and recompile

---------

Co-authored-by: Bruce Wayne <harald.the.engineer@gmail.com>
Co-authored-by: YassineYousfi <yyousfi1@binghamton.edu>
---
 openpilot/selfdrive/modeld/models/big_driving_tinygrad.pkl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/openpilot/selfdrive/modeld/models/big_driving_tinygrad.pkl b/openpilot/selfdrive/modeld/models/big_driving_tinygrad.pkl
index 48a94630ebabfe..0d35bf3f86c0ef 100644
--- a/openpilot/selfdrive/modeld/models/big_driving_tinygrad.pkl
+++ b/openpilot/selfdrive/modeld/models/big_driving_tinygrad.pkl
@@ -1,3 +1,3 @@
 version https://git-lfs.github.com/spec/v1
-oid sha256:51420a138242fe57dc4c2d2530ae8edb74afbd46f3282be8995ee3b603b79c04
-size 799942038
+oid sha256:256f2178366479a211933701c03518621ed3970289e6a8a43e5a2bcb15e55b5f
+size 799942179
